services/blockstorage/block_sync_server_test.go

```diff
@@ -1,44 +0,0 @@
-package blockstorage
-
-import (
-	"context"
-	"github.com/orbs-network/orbs-network-go/instrumentation/log"
-	"github.com/orbs-network/orbs-network-go/test/builders"
-	"github.com/orbs-network/orbs-spec/types/go/services/gossiptopics"
-	"github.com/stretchr/testify/require"
-	"os"
-	"sync"
-	"testing"
-)
-
-func TestAvailabilityRequestWithNilOrInvalidPayloadDoesNotPanic(t *testing.T) {
```

Comment:
where are the nil tests now?

Reply:
or are we assuming gossip cannot send invalid data?

Reply:
this is gossip's repsonsibility